pkg/operator/operator.go

```diff
@@ -70,5 +80,10 @@ func (o *Operator) AddControllers() error {
 
 // Start starts the operator's controller-manager.
 func (o *Operator) Start() error {
-	return o.manager.Start(signals.SetupSignalHandler())
+	stopCh := signals.SetupSignalHandler()
+
+	// Report status to the CVO.
+	go o.status.Report(stopCh)
```

Comment:
to clarify, this will report to the CVO may be a "Fail" but  the controller the will run anyway?

Reply:
Yeah, the controller will run but may report as failing with a message about dependencies not being ready if the machine-api-operator has not reported as ready. It will poll and eventually report as available / not failing if the dependencies become available though.

It will also continue working, or trying to, if there's a problem reporting to the CVO. So it still works on vanilla Kubernetes.